### PR TITLE
Add gpt2-medium-chinese model card

### PR DESCRIPTION
**Pull request by mymusise**
Hi,

this PR adds the model card for **gpt2-medium-chinese** model

### CODE REVIEW

File: model_cards/mymusise/gpt2-medium-chinese/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+language: zh
+---
+
+# gpt2-medium-chinese
+
+
+# Overview
+
+- **Language model**: GPT2-Medium
+- **Model size**: 1.2G
```
Comment (julien-c) [inline model_cards/mymusise/gpt2-medium-chinese/README.md:11]:
```suggestion
- **Model size**: 1.2GiB 
```

Would be interesting to have number of params too

File: model_cards/mymusise/gpt2-medium-chinese/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+text_generator = TextGenerationPipeline(model, tokenizer)
+print(text_generator("今日", max_length=64, do_sample=True, top_k=10))
+print(text_generator("跨越山丘", max_length=64, do_sample=True, top_k=10))
+```
+输出
+```text
+[{'generated_text': '今日 ， 他 的 作 品 也 在 各 种 报 刊 发 表 。 201 1 年 ， 他 开 设 了 他 的 网 页 版 《 the dear 》 。 此 外 ， 他 还 在 各 种 电 视 节 目 中 出 现 过 。 2017 年 1 月 ， 他 被 任'}]
+[{'generated_text': '跨越山丘 ， 其 中 有 三 分 之 二 的 地 区 被 划 入 山 区 。 最 高 峰 是 位 于 山 脚 上 的 大 岩 （ ） 。 其 中 的 山 脚 下 有 一 处 有 名 为 的 河 谷 ， 因 其 高 度 在 其 中 ， 而 得 名 。'}]
+```
```
Comment (julien-c) [inline model_cards/mymusise/gpt2-medium-chinese/README.md:33]: if needed you can specify sample inputs (right now you'll get the defaults on your model page), see https://huggingface.co/docs#how-can-i-control-my-models-widgets-example-inputs

**Comment by julien-c** [comment]
The Inference API is failing to load your model so there might be a configuration issue:

Screenshot 2020-11-08 at 11 02 01

I suspect this line is not working: `tokenizer = AutoTokenizer.from_pretrained("mymusise/gpt2-medium-chinese")`

What kind of tokenizer are you using?

**Comment by julien-c** [comment]
(cc @Narsil @JetRunner)

**Comment by JetRunner** [comment]
Well many models actually use BERT Chinese's tokenizer so maybe that's the case. If so, how can we combine the BERT tokenizer with GPT model? (I think we've handled similar situations before)

File: model_cards/mymusise/gpt2-medium-chinese/README.md (hunk 2 above)
Comment (mymusise) [reply model_cards/mymusise/gpt2-medium-chinese/README.md:33]: OK, thanks

**Comment by mymusise** [comment]
> The Inference API is failing to load your model so there might be a configuration issue:
> 
> Screenshot 2020-11-08 at 11 02 01
> 
> I suspect this line is not working: `tokenizer = AutoTokenizer.from_pretrained("mymusise/gpt2-medium-chinese")`
> 
> What kind of tokenizer are you using?

Yes, I use BertTokenizer, but how to change the Tokenizer and Model class of the default example?
```
from transformers import AutoTokenizer, TFAutoModelWithLMHead

tokenizer = AutoTokenizer.from_pretrained("mymusise/gpt2-medium-chinese")

model = TFAutoModelWithLMHead.from_pretrained("mymusise/gpt2-medium-chinese")
```
I specify `architectures=["TFGPT2LMHeadModel"]` on my configs, but it didn't show in the model card.

Is there any document I can reference?

**Comment by mymusise** [comment]
> Well many models actually use BERT Chinese's tokenizer so maybe that's the case. If so, how can we combine the BERT tokenizer with GPT model? (I think we've handled similar situations before)

Yes, I have tried to use `GPT2Tokenizer` by specific `vocab_file` and `merges_file`, but it didn't work.
```
tokenizer = GPT2Tokenizer(vocab_file=my_vocab_file, merges_file=my_merges_file)
```
Is there any examples about how to custom GPT2Tokenizer that I can reference? Many thanks.

**Comment by JetRunner** [comment]
> > Well many models actually use BERT Chinese's tokenizer so maybe that's the case. If so, how can we combine the BERT tokenizer with GPT model? (I think we've handled similar situations before)
> 
> 
> 
> Yes, I have tried to use `GPT2Tokenizer` by specific `vocab_file` and `merges_file`, but it didn't work.
> 
> ```
> 
> tokenizer = GPT2Tokenizer(vocab_file=my_vocab_file, merges_file=my_merges_file)
> 
> ```
> 
> Is there any examples about how to custom GPT2Tokenizer that I can reference? Many thanks.

cc @julien-c

**Comment by julien-c** [comment]
Yes, see the doc here: https://huggingface.co/transformers/master/main_classes/configuration.html – look for `tokenizer_class`

In config.json:

> The name of the associated tokenizer class to use (if none is set, will use the tokenizer associated to the model by default).

**Comment by mymusise** [comment]
> Yes, see the doc here: https://huggingface.co/transformers/master/main_classes/configuration.html – look for `tokenizer_class`
> 
> In config.json:
> 
> > The name of the associated tokenizer class to use (if none is set, will use the tokenizer associated to the model by default).


Thanks julien, I have add `tokenizer_class="BertTokenizer"` to the [configs](https://s3.amazonaws.com/models.huggingface.co/bert/mymusise/gpt2-medium-chinese/config.json) and update it. But the web page still using `AutoTokenizer` as example. Cache?

**Comment by julien-c** [comment]
The webpage will still display `AutoTokenizer` – but it will now work.

**Comment by julien-c** [comment]
As you can see the Inference widget now works: https://huggingface.co/mymusise/gpt2-medium-chinese?text=%E6%88%91%E5%8F%AB%E6%9C%B1%E5%88%A9%E5%AE%89%EF%BC%8C%E6%88%91%E5%96%9C%E6%AC%A2

**Comment by JetRunner** [comment]
@julien-c I think many Chinese models in our hub have the same problem. I'll have them fixed.

**Pull request merged**